build.gradle

```diff
@@ -65,7 +65,9 @@ subprojects {
         compile 'com.datastax.cassandra:cassandra-driver-core:3.1.0'
         compile 'com.google.guava:guava:18.0'
         compile 'joda-time:joda-time:2.8.2'
-        compile 'org.elasticsearch:elasticsearch:1.7.6'
+        compile 'org.elasticsearch.client:elasticsearch:6.4.3'
```

Comment:
This should be changed to org.elasticsearch:elasticsearch:6.4.3
Ref) Compile Dependencies section found in https://mvnrepository.com/artifact/org.elasticsearch.client/elasticsearch-rest-high-level-client/6.4.3
And how about using 6.8.3 instead of 6.4.3 to match with version of ES?

Reply:
Yes, Graphene Already had used a reference to a 6.4.3 Elasticsearch version. 

I agree that match to ES version and client. But when using the client 6.8.3, compile error invokes. I guess that Elasticsearch's client implementation in 6.8.3 does not consider 6.4.3 backward compatibility.

I will work on changing the ES Client Version later.